yoyodyne/evaluators.py

```diff
@@ -331,5 +331,5 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
         action=util.UniqueAddAction,
         choices=_eval_factory.keys(),
         default=defaults.EVAL_METRICS,
-        help="Additional metrics to compute. Default: none.",
+        help="Additional metrics to compute.",
```

Comment:
To be sure, does this always add to the default, or if we set only one eval_metric, does it override the default. The help language makes it sound like the former

Reply:
It's the former. The "default metric" as you're calling it is set by `--checkpoint_metric` in my design and can actually include all three metrics we support (loss, accuracy, SER).